activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/model/IntegrationEvent.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.connectors.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class IntegrationEvent {
+
+    private String processInstanceId;
+
+    private String processDefinitionId;
+
+    private String correlationId;
```

Comment:
@erdemedeiros we will need to remove this one.

Reply:
ok